HerPublicWebsite.BusinessLogic/Services/EligiblePostcode/EligiblePostcodeService.cs

```diff
@@ -10,11 +11,32 @@ public interface IEligiblePostcodeService
 
 public class EligiblePostcodeService : IEligiblePostcodeService
 {
+    // build structure means json file is in different location when running locally
+    private const string PostcodeJsonPath = "Services/EligiblePostcode/EligiblePostcodeData.json";
+
+    private const string LocalPostcodeJsonPath =
+        "../HerPublicWebsite.BusinessLogic/Services/EligiblePostcode/EligiblePostcodeData.json";
```

Comment:
Reminder to come back to this if we still need to later: it feels brittle to have the paths hard-coded here and I think we'd want a better way to tell if we're running the app locally than checking if the file exists at that path.

Reply:
this should now be resolved; resources are loaded in the same way between local and deployed